lampetia-security-model/src/main/scala/lampetia/security/model/models.scala

```diff
@@ -279,7 +298,7 @@ trait SecurityModel {
       }
       object parentResource extends Composite[Option[Resource]] {
         val resourceId =
-          property[Option[ResourceId]]("resourceId")
```

Comment:
since parent resource is optional, resourceId don't need to be optional too. right?